app/js/init.js

```diff
@@ -2,8 +2,10 @@
 
 var React = require('react');
 var Player = require('./player');
+var simpleStream = require('simple-stream');
+var Stream = simpleStream.Stream;
 
 React.render(
-  <Player />,
+  <Player isFollowingStream={Stream()} broadcastNameStream={Stream()} isBroadCastingStream={Stream()} />,
```

Comment:
I'd prefer these properties not be set on the Player directly. There should be an API for interacting with the player which controls the state of playback. I'll try and spec that out tonight.